x-pack/plugins/security_solution/server/lib/detection_engine/signals/types.ts

```diff
@@ -298,6 +301,8 @@ export interface SearchAfterAndBulkCreateParams {
   enrichment?: SignalsEnrichment;
   bulkCreate: BulkCreate;
   wrapHits: WrapHits;
+  trackTotalHits?: boolean;
+  sortOrder?: SortOrderOrUndefined;
```

Comment:
It would be great to instead use `estypes.SearchSortOrder` here, to keep this decoupled from the validation schema as much as possible.